_field-types/index.md

```diff
@@ -88,6 +82,41 @@ POST sample-index1/_mapping
 You cannot change the mapping of an existing field, you can only modify the field's mapping parameters. 
 {: .note}
 
+## Mapping parameters
+
+Mapping parameters are used to configure the behavior of fields in an index. The following table lists commonly used mapping parameters.
+
+Parameter | Description
+:--- | :---
+`analyzer` | Specifies the analyzer used to analyze string fields.
+`boost` | Specifies a field-level query time to boost.
+`coerce` | Tries to convert the value to the specified data type.
+`copy_to` | Copies the values of this field to another field.
+`doc_values` | Specifies whether the field should be stored on disk to make sorting and aggregation faster.
+`dynamic` | Determines whether new fields should be added dynamically.
+`enabled` | Specifies whether the field is enabled or disabled. 
+`format` | Specifies the date format for date fields. 
+`ignore_above` | Skips indexing values that are longer than the specified length.
+`ignore_malformed` | Specifies whether malformed values should be ignored. 
+`index` | Specifies whether the field should be indexed.
+`index_options` | Specifies what information should be stored in the index for scoring purposes.
```

Comment:
do you think it will be helpful to add possible values for each of these parameters (or examples in case the value set is not limited)

Reply:
@mgodwan Yes to both.

Reply:
What are the default values?

Reply:
The default values for each of these may vary across different field types (e.g. keyword, text may have different defaults for different parameters)

For each field type documentation (e.g. text, integer, geopoint), we can then share what the default/allowed values for the parameters are.

Reply:
@mgodwan Please verify the default and allowed values are accurate and make any necessary changes, as needed. Thank you.